yarn.lock

```diff
@@ -1955,19 +2052,56 @@
     "@file-services/path" "^7.0.1"
     "@file-services/types" "^7.0.1"
 
+"@wixc3/common@^3.0.4":
```

Comment:
there's a duplicate @wixc3/common now, not sure why

Reply:
Might be a missed upgrade, let's revisit after https://github.com/wixplosives/engine/pull/1520 is merged